### PR TITLE
[5263][3.1.x] Video Center View All videos for a channel does not reflect the total amount of videos

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5263

### CODE REVIEW

**Comment by jvega190** [comment]
Build update pending (waiting for approval)

File: sources/app/src/components/Cards/Cards.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ class Cards extends Component {
           [category.sort.by]: category.sort.order
         };
       }
+      const size = category.numResults ?? 6;
```
Comment (rart) [inline sources/app/src/components/Cards/Cards.js:59]: Why is there a hardcoded 6 here?

Comment (jvega190) [reply sources/app/src/components/Cards/Cards.js]: Updated so it won't send size parameter in query if not set (so it'll return the default value from service)

**Review by rart — APPROVED** [review]

**Comment by jvega190** [comment]
@sumerjabri done

**Comment by jvega190** [comment]
@sumerjabri I pushed the build update since the changes are approved now

**Pull request merged**